common/thorhelper/thorfile.cpp

```diff
@@ -78,7 +78,7 @@ class DiskWorkUnitReadArg : public CThorDiskReadArg
     }
     virtual unsigned getFlags() override
     {
-        return TDRnocrccheck;
+        return 0;
```

Comment:
Was this added at some point to default row crc checking to off?
If so - it didn't really - and we still have on by default in Thor.. 
Therefore should we change checkFileCrc default to off in Thor in this PR too?